app/controllers/feedbacks_controller.rb

```diff
@@ -75,8 +75,8 @@ class FeedbacksController < ApplicationController
   end
 
   post '/feedbacks/edit/:id' do
```

Comment:
This route should be PUT '/feedbacks/:id' make sure that your form has a hidden input field with a `value="PUT"` and a `name="_method"`